charts/jaeger-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.17.0
+version: 2.18.2
```

Comment:
why you bump to 2.18.2 and not just 2.18.0?

Reply:
I had some issues on my own fork when running GitHub Actions 🤦 I am sorry for that! Should I switch to 2.18.0?

Reply:
yes please

Reply:
No problem! 😄 Done!